WooCommerce/Classes/ViewModels/Order Details/OrderDetailsViewModel.swift

```diff
@@ -108,16 +108,6 @@ final class OrderDetailsViewModel {
         }
     }
 
-    /// Indicates if payment for the order can be collected using an external card reader
```

Comment:
This was probably lingering from a previous PR. It is not used anywhere, so of it goes